test/Generator.Tests/TestDtdlModels/Asset.json

```diff
@@ -0,0 +1,53 @@
+{
+  "@id": "dtmi:test:Asset;1",
+  "@type": "Interface",
+  "@context": "dtmi:dtdl:context;2",
+  "displayName": "Address",
```

Comment:
Asset

Reply:
Good eye. Updated.

(Letting you resolve this conversation since that's the equivalent of closing in ADO.)